common/oauthTokenManager.go

```diff
@@ -343,8 +440,23 @@ func (uotm *UserOAuthTokenManager) HasCachedToken() (bool, error) {
 	return uotm.credCache.HasCachedToken()
 }
 
-// RemoveCachedToken delete all the cached token.
+// RemoveCachedToken deletes the token and the AuthenticationRecord as part of the logout command execution.
+// Returns an error if encountered during file deletion or checking.
 func (uotm *UserOAuthTokenManager) RemoveCachedToken() error {
+	filePath := filepath.Join(AzcopyJobPlanFolder, defaultAuthFileName)
+
+	// Check if the file exists
+	if _, err := os.Stat(filePath); err == nil {
+		// File exists, so delete it
+		if err := os.Remove(filePath); err != nil {
+			return fmt.Errorf("error deleting AuthenticationRecord: %v", err)
+		}
+		fmt.Println("AuthenticationRecord deleted successfully.")
+	} else if os.IsNotExist(err) {
```

Comment:
both these else cases can be removed as we need to clear credcache after deleting record file.

Reply:
Done in the latest commit.